packages/api/cms-api/src/user-permissions/utils/getDate.ts

```diff
@@ -0,0 +1,7 @@
+export function getDate(str?: string | number | Date) {
```

Comment:
I'd prefer if we use date-fns for this

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/a89b03838f75152ac5c311d1a9722869103529ab